src/index.ts

```diff
@@ -169,10 +169,13 @@ function nuxtStorybookOptions (options) {
   if (!srcDir.startsWith('/')) {
     srcDir = path.resolve(options.rootDir, srcDir)
   }
-  nuxtStorybookConfig.stories = [
-    '~/components/**/*.stories.@(ts|js)',
-    ...nuxtStorybookConfig.stories
-  ].map(story => upath.normalize(story
+
+  const storiesDir = path.resolve(options.rootDir, 'components/**/*.stories.@(ts|js)')
+  if (fsExtra.existsSync(storiesDir)) {
```

Comment:
Shouldn't we check existence without **/stories in the path?

Reply:
PS: ~ alias (default) refers to srcDir not rootDir

Reply:
@pi0 Indeed, I've updated!